composer.json

```
@@ -16,18 +16,18 @@
         }
     ],
     "require": {
-        "php": ">=7.3.0,<7.5.0",
-        "doctrine/common": "^2.6",
-        "doctrine/dbal": "^2.5",
-        "doctrine/migrations": "^2.2",
+        "php": ">=7.4.0",
+        "doctrine/common": "^2.6 || ^3.0",
+        "doctrine/dbal": "^2.5 || ^3.0",
+        "doctrine/migrations": "^3.0",
         "doctrine/orm": "^2.5",
         "psr/container": "^1.0.0"
     },
     "require-dev": {
         "doctrine/coding-standard": "^7.0",
```

I think it should be upgraded to satisfy composer requirements
```suggestion
        "doctrine/coding-standard": "^8.1",
```

---

I can confirm, after testing locally, that this would solve the failing Travis build.

@edigu could you please accept this suggested change ?